public/index.php

```diff
@@ -10,6 +10,10 @@
   <li>
     <a href="discussions.php"><strong>View</strong></a> - list all discussions
   </li>
+  <!-- Remove this -->
+  <li>
+    <a href="marked_entity_files.php?marked_entity_id=1"><strong>View</strong></a> - list all marked entity files
```

Comment:
Should this be removed? (according to your remove this comment above it :P)

Reply:
😂 I think I was typing something and then I got interrupted and forgot about this,  I meant to say something along the lines of "remove this when marked entity page is ready"